playbooks/group_vars/webportals.yml

```diff
@@ -118,22 +123,23 @@ webportal_server_config_defaults:
   sia_api_port: 9980
   sia_api_password: "{{ lookup('pipe','openssl rand -base64 32') }}"
   hsd_api_key: "{{ lookup('pipe','openssl rand -base64 32') }}"
+  server_uid: "{{ lookup('pipe','openssl rand -hex 8') }}"
+  accounts_test_user_email: "{{ lookup('pipe','openssl rand -hex 8') }}@example.com"
+  accounts_test_user_password: "{{ lookup('pipe','openssl rand -hex 8') }}"
 
-webportal_cluster_config_defaults:
+webportal_common_config_defaults:
```

Comment:
Should we have a comment explaining why this is `common` not `cluster`?

Reply:
Added a comment.